src/resonanceReconstruction/rmatrix/src/horner.hpp

```diff
@@ -0,0 +1,35 @@
+/** @brief Horner evaluation of a polynomial function for different x and y
+ *         types
+ *
+ *  A polynomial is defined as a sequence of coefficients c_i so that the
+ *  polynomial of order n is given by:
+ *
+ *    y = c_n * x^n + c_(n - 1) * x^(n - 1) + c_(n - 2) * x^(n - 2) + ...
+ *
+ *  With the Horner scheme, a polynomial is evaluated as follows:
+ *
+ *    y = c_n * (x + c_(n-1) * (x + c_(n - 2) * (x + ...
+ *
+ *  The main reason for using the Horner scheme is computational efficiency.
```

Comment:
I wonder if it's more accurate as well. I'd have to work it out, or look it up.

Reply:
Good question, I wouldn't know the answer to that either.